models/topics.models.js

```diff
@@ -1,7 +1,17 @@
 const db = require('../db/connection.js')
 
-exports.retrieveAllTopics = () => {
-    return db.query('SELECT * FROM topics;').then(({ rows: topics }) => {
+exports.retrieveAllTopics = (topic) => {
```

Comment:
rejections could take place here if topic was undefined or if you invoked this function with req.body you could check if the body was {} OR {topic: mitch} OR {invalidTopic: mitch} and deal with it accordingly.